PWGDQ/Core/CutsLibrary.cxx

```diff
@@ -392,13 +392,6 @@ AnalysisCompositeCut* o2::aod::dqcuts::GetCompositeCut(const char* cutName)
     return cut;
   }
 
-  if (!nameStr.compare("emu_electronCuts")) {
-    cut->AddCut(GetAnalysisCut("jpsiStandardKine"));
-    cut->AddCut(GetAnalysisCut("electronStandardQualityForO2MCdebug4"));
-    cut->AddCut(GetAnalysisCut("electronPIDnsigmaSkewed"));
-    return cut;
-  }
-
```

Comment:
Why remove this?

Reply:
Dear @iarsene, Sorry for the inconvinience. I created this PR 5 days before. Are these cuts are added recently ? In my local O2Physics version these cuts are not there in CutLibrary.cxx. I have not removed anything, I simply added few extra cuts in this PR. Anyway I will close this PR and create a new one. Again sorry for the inconvinience.

Reply:
Hii, I saw these cuts are added recently. In the PR https://github.com/AliceO2Group/O2Physics/pull/6925

Reply:
Since the PR  https://github.com/AliceO2Group/O2Physics/pull/6925 is merged after https://github.com/AliceO2Group/O2Physics/pull/7140 is created.That's why it shows removal of these cuts